examples/controlnet/train_controlnet_sdxl.py

```diff
@@ -103,18 +102,22 @@ def log_validation(vae, unet, controlnet, args, accelerator, weight_dtype, step)
         )
 
     image_logs = []
-
+    negative_prompt = "anime, low quality, bad quality, sketches, cartoon, graphic, text, painting, crayon, graphite, abstract, glitch, deformed, mutated, ugly, disfigured"
```

Comment:
I don't think this is related to the PR no? Could we maybe remove this extra line?